plugins/modules/azure_rm_aduser_info.py

```diff
@@ -224,14 +235,22 @@ def exec_module(self, **kwargs):
             elif self.all:
                 # this returns as a list, since we parse multiple pages
                 ad_users = asyncio.get_event_loop().run_until_complete(self.get_users())
-
             self.results['ad_users'] = [self.to_dict(user) for user in ad_users]
 
         except Exception as e:
             self.fail("failed to get ad user info {0}".format(str(e)))
 
         return self.results
 
+    def on_premises_extension_attributes_to_dict(self, on_premises_extension_attributes):
+        extension_attributes = {}
+        for index in range(1, 16 + 1):
+            attribute_name = f'extension_attribute{index}'
+            if hasattr(on_premises_extension_attributes, attribute_name):
+                attr_value = getattr(on_premises_extension_attributes, attribute_name)
+                if attr_value is not None:
+                    extension_attributes[attribute_name] = attr_value
+        return extension_attributes
```

Comment:
expected 1 blank line, found 0
```suggestion
        return extension_attributes

```